data-fr.json

```diff
@@ -1 +1 @@
ontinuer à prendre en charge en toute sécurité les patients et protéger son personnel soignant dans la durée, la clinique La Parisière à Bourg-de-Péage lance elle aussi un appel aux dons pour récupérer du matériel de protection. Voici la liste de leurs besoins : masques (chirurgicaux, FFP1, FFP2, FFP3), gants (Nitrile, Latex, Vinyl), blouses, surblouses, combinaisons à usage unique (non-tissé ou plastique), tabliers plastique à usage unique, charlottes, coiffes, cagoules à usage unique. Vous pouvez joindre la clinique au 04 75 70 81 18. ","FFP2 masks (N95) or FFP3, Surgical masks (type II, IIR), Face shields, Disposable booties, Gloves, Gowns","Yes","","45.040256","5.0566503",26],["x","fr","23","4/4/2020 13:48:08","Add a new donation site","Hôpitaux Drôme Nord Site Romans-sur-Isère Service des Urgences","607 Avenue Geneviève de Gaulle-Anthonioz, 26100 Romans-sur-Isère, France","Rue Geneviève Anthonioz de Gaulle, 26260 Saint-Donat-sur-l'Herbasse, France","Romans-sur-Isère","26 Drôme","Les hôpitaux de Valence et de Romans-sur-Isère appellent à la générosité pour protéger leurs soignants. À Romans, les surblouses manquent cruellement. Toute entreprise, collectivité ou particulier disposant de surblouses à manches longues ou de combinaisons à manches longues à usage unique peut contacter les Hôpitaux Drome Nord au 04 75 05 75 62.","Gloves, Gowns","No","","45.121652","4.9759673",27],["x","fr","24","4/4/2020 13:57:55","Add a new donation site","Centre Hospitalier de Saint Flour"," 2 avenue du Dr Mallet - 15100 SAINT FLOUR","2 Ave du Dr Louis Mallet, 15100 Saint-Flour, France","Saint-Flour","15 Cantal","","FFP2 masks (N95) or FFP3, Surgical masks (type II, IIR), Face shields, Disposable booties, Gloves","Yes","","45.0333001","3.0868379",28],["x","fr","25","4/4/2020 14:02:41","Add a new donation site","Hôpital privé Saint-Camille","2 Rue des Pères Camilliens, 94360 Bry-sur-Marne, France","2 Rue des Pères Camilliens, 94360 Bry-sur-Marne, France","Bry-sur-Marne","94 Val de Marne","","FFP2 masks (N95) or FFP3, Surgical masks (type II, IIR), Face shields, Safety goggles, Disposable booties, Gloves, Gowns, Hand sanitizer, Disinfecting wipes, Thermometers","Yes","","48.8338774","2.5279742",29],["x","fr","26","4/4/2020 14:14:11","Add a new donation site","Centre Hospitalier de Chauny","94 Rue Anciens Combattants 02300 CHAUNY","94 Square des Anciens Combattants, 02300 Chauny, France","Chauny","02 Aisne","","FFP2 masks (N95) or FFP3, Surgical masks (type II, IIR), Face shields, Disposable booties, Gloves","Yes","","49.6194154","3.2064971",30],["x","fr","27","4/4/2020 14:19:59","Add a new donation site","Centre Hospitalier de Compiègne","ZAC de Mercières 3, 8 Avenue Henri Adnot, 60200 Compiègne, France","ZAC de Mercières 3, 8 Avenue Henri Adnot, 60200 Compiègne, France","Compiègne","62 Pas-de-Calais","","FFP2 masks (N95) or FFP3, Surgical masks (type II, IIR), Safety goggles, Disposable booties, Gloves, Gowns, Hand sanitizer","No","","49.3868474","2.7902822",31]]}
```

Comment:
How are you planning on fetching these? Would it happen on each deploy or will we be hitting firebase directly?

Reply:
I'm hoping to hit Firebase or GCP buckets directly